kpet/utils.py

```diff
@@ -33,3 +37,106 @@ def get_template_content(filename, dbdir=None):
     template_path = get_template_path(filename, dbdir)
     with open(template_path) as file_handler:
         return file_handler.read()
+
+
+def print2(s):
+    sys.stderr.write(s + '\n')
+
+
+def smart_layout_insert_prefix(f_layout, obj):
+    if obj is None:
+        return None
+    prefix = os.path.dirname(os.path.realpath(f_layout))
+    return '%s/%s' % (prefix, str(obj))
+
+
+def smart_get_layout(f_layout):
```

Comment:
I think that you can remove this function and use `json.load` instead. Less code to maintain :smile:. Please note it's `json.load` instead of `json.loads`. Quoting a fragment of the doc
```
Signature: json.load(fp, encoding=None, cls=None, object_hook=None, parse_float=None, parse_int=None, parse_constant=None, object_pairs_hook=None, **kw)
Docstring:
Deserialize ``fp`` (a ``.read()``-supporting file-like object containing
a JSON document) to a Python object.

If the contents of ``fp`` is encoded with an ASCII based encoding other
than utf-8 (e.g. latin-1), then an appropriate ``encoding`` name must
be specified. Encodings that are not ASCII based (such as UCS-2) are
not allowed, and should be wrapped with
``codecs.getreader(fp)(encoding)``, or simply decoded to a ``unicode``
object and passed to ``loads()``
```